label_studio/io_storages/azure_blob/models.py

```diff
@@ -122,20 +122,28 @@ def iterkeys(self):
                 continue
             yield file.name
 
-    def get_data(self, key):
+    def get_data(self, key) -> list[dict]:
         if self.use_blob_urls:
             data_key = settings.DATA_UNDEFINED_NAME
-            return {data_key: f'{self.url_scheme}://{self.container}/{key}'}
+            return [{data_key: f'{self.url_scheme}://{self.container}/{key}'}]
 
         container = self.get_container()
         blob = container.download_blob(key)
         blob_str = blob.content_as_text()
         value = json.loads(blob_str)
-        if not isinstance(value, dict):
+        if isinstance(value, dict):
+            return [value]
+        elif isinstance(value, list):
+            for idx, item in enumerate(value):
+                if not isinstance(item, dict):
+                    raise ValueError(
```

Comment:
Unfortunately, I don't think this message will actually reach the user. I think the better behavior may be to log the exception (so we see it in sentry) and import anything we can

Reply:
Worth checking w/ Sheree if she has an opinion here though tbh

Reply:
Check out the existing implementation for s3 - some storages raise an error here and some don't, I chose to add the same error to all of them to keep it consistent. I could be missing some storage-specific logic outside this function that means they're handled differently though.